gpii/node_modules/contextManager/src/ContextManager.js

```diff
@@ -160,21 +160,21 @@ gpii.contextManager.stopTemporalEnvironmentReporter = function (that) {
 gpii.contextManager.evaluateConditions = function (that, lifecycleManager, context) {
     // find logged in users
     var activeSessions = lifecycleManager.getActiveSessionTokens();
-    // if none is logged in, do nothing
+    // if noone is logged in, do nothing
     if (activeSessions.length === 0) {
         return;
     }
     var activeSession = lifecycleManager.getSession(activeSessions);
-    var newActiveContexts = gpii.contextManager.utils.findActiveContexts(context, activeSession.matchMakerOutput);
+    var newActiveContexts = gpii.contextManager.utils.findActiveContexts(context, activeSession.model.matchMakerOutput);
     fluid.log("contextManager: New active contexts: " + newActiveContexts);
     if (newActiveContexts[0] === activeSession.activeContextName) {
-        fluid.log("contextManager: Same context as before (" + activeSession.appliedContext + ") so doing nothing");
+        fluid.log("contextManager: Same context as before (" + activeSession.model.appliedContext + ") so doing nothing");
         return;
     }
-    var activeConfiguration = activeSession.matchMakerOutput.inferredConfiguration[newActiveContexts[0]];
-    var lifecycleInstructions = that.transformer.configurationToSettings(activeConfiguration, activeSession.solutionsRegistryEntries);
+    var activeConfiguration = activeSession.model.matchMakerOutput.inferredConfiguration[newActiveContexts[0]];
```

Comment:
When it is ready to be merged, could you please close and regenerate this pull request so that it doesn't include diffs from other work - thanks